spec/pluto/operation/vertical_blur_spec.cr

```diff
@@ -1,45 +1,28 @@
 require "../../spec_helper"
 
```

Comment:
This spec is the one I did a "full" conversion with. Things of note:

Spectator will create a full on class for the specs, so we can define spec specific helper methods within the `describe`, _or_ include a common set of test util methods using `include SpecHelper` (or whatever module contains said util methods).

The `let(var) { ... }` syntax defines a lazily initiated, cached variable available to all specs in the same `describe` or below. If a spec doesn't use one of these variables, it's never initialized, and once initialized, the same instance will be re-used for all calls to `var` in the same spec (so self-modifying methods would still have their changes).

Since all of these specs followed the pattern of 1) initialize image, 2) run transformation method, and 3) check digest of result, I let the `let` take care of **1** and do a one liner (with helper methods) for **2** and **3**.

Reply:
Is there a way to have different instances? Wouldn't want one spec to influence another.

Reply:
Ah, I see. Thanks for clarifying!